src/nxdoc/web-ui/web-ui-tutorials/web-ui-configuration-properties.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 'HOWTO: Configure Web UI global settings'
+description: Learn how to configure global Web UI settings.
+review:
+    comment: ''
+    date: '2019-07-19'
+    status: ok
+toc: true
+details:
+    howto:
+        excerpt: Learn how to configure global Web UI settings.
+        level: Advanced
+        tool: code
+        topics: Web UI
+labels:
+    - nuxeo-web-ui
+    - extension
+    - customization
+    - tutorial
+tree_item_index: 650
+
+---
+
+{{! excerpt}}
+Nuxeo Web UI provides a simple way to configure global settings in order to have consistent behaviour across the platform. These configurations are handled as a configuration extension.
+
+For more details please take a look to [HOWTO: Contribute to an Extension]({{page page='how-to-contribute-to-an-extension'}}).
+{{! /excerpt}}
+
+## Default Settings
+
+The XML contribution for the configuration extension is located [here](https://github.com/nuxeo/nuxeo-web-ui/blob/master/plugin/web-ui/addon/src/main/resources/OSGI-INF/web-ui-properties.xml) and contains a set predefined default settings using `org.nuxeo.web.ui` namespace.
+
+For the moment, the default settings include:
+
+### Enrichers
```

Comment:
We could benefit a cross-page reference on https://doc.nuxeo.com/nxdoc/web-ui-performance/